lifechanger/package.json

```diff
@@ -19,6 +19,7 @@
     "@angular/platform-browser-dynamic": "^17.3.0",
     "@angular/router": "^17.3.0",
     "rxjs": "~7.8.0",
+    "swiper": "^11.1.1",
```

Comment:
is it required change ? If no - please remove it

Reply:
Thank you, dear Mentor, for the remark, I repaired:)